File: test/fixtures/php-duplicates/invoice.php
<?php
namespace Billing;

function invoiceTotal(array $lines, float $taxRate): float
{
    $total = 0.0;
    foreach ($lines as $line) {
        $total += $line['price'] * $line['quantity'];
    }
    $tax = round($total * $taxRate, 2);
    return $total + $tax;
}

File: test/fixtures/rust-duplicates/src/invoice.rs
use crate::Order;

pub fn invoice_total(order: &Order) -> f64 {
    let subtotal: f64 = order.lines.iter().map(|line| line.price * line.quantity as f64).sum();
    let discount = if order.loyal { subtotal * 0.1 } else { 0.0 };
    subtotal - discount + order.shipping
}
